modules/preview-react/side-panel/stories/SidePanel.stories.mdx

```diff
@@ -112,7 +110,7 @@ is an example:
 
 #### Props
 
-<ArgsTable of={SidePanel} />
+<SymbolDoc name="SidePanel" fileName="/preview-react/" />
```

Comment:
I noticed with `fileName` we have a mix of using leading and trailing slashes and not. I didn't notice any difference in local testing. Just heads up it will have other developers wondering which convention to use, and what the impact might be.

Reply:
I've added slashes and `fileName` to every call of `<SymbolDoc>`